frontend/src/logic/schedule-logic/public-holidays.logic.ts

```diff
@@ -0,0 +1,70 @@
+// dates as represented here follow [day, month] pattern
+// months as used here are human-readable i.e. go from 1 to 12 (hence + 1)
+
+export class PublicHolidaysLogic {
+  private publicHolidayDates: number[][];
```

Comment:
Zamiast przechowywać to jako dwuwymiarową tablicę, można by tutaj stworzyć typ Date coś ala:
`type Date = [day: number, month: number]`
i zrobić jego jednowymiarowy odpowiednik. Dzięki temu nie potrzebowalibyśmy komentarza u góry :)